lib/views/general/_frontpage_videos.html.erb

```diff
@@ -0,0 +1,33 @@
+<% 
+    @research_videos = [
+    { title: 'Reforming Freedom of Information: mySociety policy paper launch event',
+        id: 'AbXowwJDn3M'},
+    { title: 'The FOI Clearing House: an openDemocracy investigation into FOI at the heart of government',
+        id: 'wXbhBHfeOzk'},
+    { title: 'Workshop: What can you do with FOI?',
+        id: 'meaLp7p2Yok'},
+    ]
+%>
+
+<div class="homepage-video-section" id="homepage-video">
+    <div class="row homepage-video-section__vs-content">
+        <div class="vs-content__headings ">
+            <h2><%= _("Recent videos") %></h2>
+        </div>
+        <div class="vs-content__video-grid col-3-grid">
+            <% @research_videos.sample(3).each do |article| %>
+                <div class="video-grid__item">
+                <img class="item-content__img" src="http://img.youtube.com/vi/<%= article[:id] %>/hqdefault.jpg" alt="<%= article[:title] %>" loading="lazy">
```

Comment:
Should this be a http**s** url? No need to fix – I can fix it in the merge, just need to know whether to tweak :)

Reply:
Hi Gareth, yes please I didn't see that. It should be https.